src/themes/masschallenge/modules/modal.overrides

```diff
@@ -19,3 +19,31 @@
 .ui.modal > .teal.segment {
   background-color:  #b8dde4 !important;
 }
+
+.ui.modal.centered {
+  margin-top: 4rem; 
+  max-width: 368px; 
+  min-width: 320px;
+}
+
+.ui.modal .scrolling.content {
+    max-height: calc(85vh - 326px);
+}
```

Comment:
I have a concern about this for future use:
The class names a fairly generic and imply this could be used in various cases, but the definition on `max-height` is very specific to the use-case of the calendar picker modal. This could be confusing if someone tried to reuse the `.scrolling.content` class pairing elsewhere. I suggest using a more specific class or id (that matches the purpose (perhaps something that references `rbc-calendar`) so it will not be confused later.